common/src/main/java/org/wso2/testgrid/common/DashboardSetup.java

```diff
@@ -57,6 +61,7 @@
             ConfigurationContext.getProperty(ConfigurationContext.ConfigurationProperties.GRAFANA_APIKEY);
     private String testplanID;
 
+
```

Comment:
DashboardSetup name is incorrect. Shall we instead say GrafanaDashboardHandler since this handles the grafana dashboard for a given test plan id?